src/cfb.rs

```diff
@@ -78,7 +80,9 @@ impl Cfb {
     pub fn new<R: Read>(mut reader: &mut R, len: usize) -> Result<Cfb, CfbError> {
         // load header
         let (h, mut difat) = Header::from_reader(&mut reader)?;
-        let mut sectors = Sectors::new(h.sector_size, Vec::with_capacity(len));
+        assert_ne!(h.sector_size, 0);
```

Comment:
Replace this assert with an error Result.

Reply:
The intent was a bit unclear, I've cleaned that up. It can't be zero because of the validation done in `Header::from_reader`.